### ISSUE TITLE
Fragmentions should scroll to img alt text

### ISSUE THREAD

**Issue by tantek** [indieweb/fragmention#4]
The [Fragmention specification](https://indieweb.org/fragmention) does not mention img element alt text as part of the text that it searches for matches, and I think it should because there are use-cases for it.

Use-case 1: I want to reference the third image in [this post](https://tantek.com/2020/004/t1/first-day-in-office-this-year) with a fragmention, and figured the alt text would provide a nice way to do so.

Use-case 2: A screen reader user is listening to a page, including the inline alt text from images, and decides to respond or otherwise comment on something they heard in the alt text, constructs a fragmention with some of that text and expects it to work like any other text phrase in the document with a fragmention.

Use-case 3: A web image search crawler is indexing images on various websites and as part of displaying search results for an image, wants to link directly to a specific image on a page that happens to have alt text which matched the search terms of the search query. In order to link directly to that image, the search engine determines a several word phrase of that img’s alt text and uses it to construct a fragmention link which is then provided in the search result.

In each of these cases, it makes sense for the user clicking on a fragmention link to be able to "scroll to" a specific image / that specific section of alt text, and thus the specification should both support that behavior and make it a MUST for interoperability and reliability.

(Originally published at: https://tantek.com/2020/006/b1/fragmentions-should-scroll-img-alt)
